cabby/geo/sample_poi.py

```diff
@@ -33,13 +33,13 @@
 
 FLAGS = flags.FLAGS
 flags.DEFINE_enum(
-    "region", None, ['Pittsburgh', 'Manhattan'],
-    "Map areas: Manhattan or Pittsburgh.")
+  "region", None, ['Pittsburgh', 'Manhattan'],
+  "Map areas: Manhattan or Pittsburgh.")
```

Comment:
Don't need to do anything here, but we clearly have a common set of options involving region and others, so we should probably refactor to pull those common flag definitions rather than redefining them for every application.